pkg/reconciler/v1alpha1/service/service.go

```diff
@@ -261,12 +261,12 @@ func (c *Reconciler) reconcileRoute(ctx context.Context, service *v1alpha1.Servi
 	logger := logging.FromContext(ctx)
 	desiredRoute, err := resources.MakeRoute(service)
 	if err != nil {
-		// This should be unreachable as configuration creation
-		// happens first in reconcile()
+		// This should be unreachable as route creation
```

Comment:
This should remain configuration, or perhaps reworded for clarity. In the reconcile function we reconcile configuration first. The validation on configuration will catch any errors in service that may cause MakeRoute to fail so if it gets to MakeRoute() it will succeed. 

I added the comment as our test coverage shows this uncovered, but it isn't possible to write a unit test on service.go without changing the logic.